Adafruit_CAP1188.h

```diff
@@ -1,63 +1,96 @@
-/*************************************************** 
-  This is a library for the CAP1188 8-Channel Capacitive Sensor
-
-  Designed specifically to work with the CAP1188 breakout from Adafruit
-  ----> https://www.adafruit.com/products/1602
-
-  These sensors use I2C/SPI to communicate, 2+ pins are required to  
-  interface
-  Adafruit invests time and resources providing this open source code, 
-  please support Adafruit and open-source hardware by purchasing 
-  products from Adafruit!
-
-  Written by Limor Fried/Ladyada for Adafruit Industries.  
-  BSD license, all text above must be included in any redistribution
- ****************************************************/
+/*!
+ *  @file Adafruit_CAP1188.h
+ *
+ *  This is a library for the CAP1188 8-Channel Capacitive Sensor
+ *
+ *  Designed specifically to work with the CAP1188 I2C/SPI 8-chan Capacitive
+ *  Sensor from Adafruit
+ *
+ *  Pick one up today in the adafruit shop!
+ *  ------> https://www.adafruit.com/product/1602
+ *
+ *  These sensors use I2C/SPI to communicate, 2+ pins are required to interface
+ *
+ *  Adafruit invests time and resources providing this open source code,
+ *  please support Adafruit andopen-source hardware by purchasing products
+ *  from Adafruit!
+ *
+ *  Limor Fried/Ladyada (Adafruit Industries).
+ *
+ *  BSD license, all text above must be included in any redistribution
+ */
 
 #if (ARDUINO >= 100)
- #include "Arduino.h"
+#include "Arduino.h"
 #else
- #include "WProgram.h"
+#include "WProgram.h"
```

Comment:
remove the if/else so we only support Arduino >= 100 now

Reply:
in all libs?

Reply:
yes its ancient, WProgram is no longer used